crates/ide/src/hover/tests.rs

```diff
@@ -3595,6 +3595,72 @@ const FOO$0: u8 = b'\x61';
 
             ---
 
+            This is a doc
+        "#]],
+    );
+    // show float literal
+    check(
+        r#"
+    /// This is a doc
+    const FOO$0: f64 = 1.0234;
+    "#,
+        expect![[r#"
+            *FOO*
+
+            ```rust
```

Comment:
It might be good to move these tests to `hir_ty::consteval::tests`

Reply:
So just have one float literal test here and move other cases to consteval?

Reply:
> So just have one float literal test here and move other cases to consteval?

Yes, that sounds good to me.

> Actually should they go in consteval this was a general problem regarding handling of floats

I think it only affects `consteval`, and current tests just test this, but if there are other problems fixed/affected by this, then adding test for those in addition to `consteval` tests is good.